### ISSUE TITLE
In what scenarios the docs about 'The name "class" must be quoted in the object...' is relevant?

### ISSUE THREAD

**Issue by alex-arriaga** [jquery/api.jquery.com#1156]
jQuery version: 3.4.1

While I was reformatting some code with an automated tool, I noticed that tool removes the quotes in this kind of code:
```javascript
// Original code
var $p = $('<p>', {
    "class": "b-info", // I am USING quotes for "class"
    text: "Appended text"
});
```

```javascript
// After passing the formatter
var $p = $('<p>', {
    class: 'b-info', // Quotes are removed from "class"
    text: "Appended text"
});
```

So, I went back to jQuery docs in https://api.jquery.com/jquery/ where it is said:

> The name **"class"** must be quoted in the object since it is a JavaScript reserved word, and **"className"** cannot be used since it refers to the DOM property, not the attribute. 

**I was wondering if you have identified scenarios/versions where this is RELEVANT**. 

I did some tests using this minimal code, in:

- Google Chrome: OK
- Safari: OK
- Firefox: OK
- MS Edge: OK
- IE 11: OK
- IE 10 (Emulation): OK
- IE 9 (Emulation): OK
- IE 8 (Emulation): FAILS
- IE 7 (Emulation): FAILS

So, **it seems, this code ONLY fails on IE 8 and older versions of that browser**,  is that correct?

```html
<!doctype html>
<html lang="en">
<body>
    <h1>jQuery test</h1>

    <p>"Appended text" will be appended. Check your console too!</p>

    <script src="./js/jquery-3.4.1.min.js"></script>
    <script>
    	(function($) {
             var $p = $('<p>', {
                 class: 'b-info', // I am not use quotes for "class"
                 text: 'Appended text'
             });

             $('body').append($p);
    	})(jQuery);
    </script>
</body>
</html>
```

**Comment by mgol** [comment]
Yeah, that's not really a useful remark anymore. I'd remove it from the docs. Do you want to submit a PR?

**Comment by dmethvin** [comment]
There are also some old tools that complain but at this point nobody should be using really old tools. 😁
